src/Utilities/copy.jl

```diff
@@ -318,14 +318,19 @@ function default_copy_to(dest::MOI.ModelLike, src::MOI.ModelLike, copy_names::Bo
             for S in single_variable_types
         ]
     else
-        vector_of_variables_not_added = [
-            copy_vector_of_variables(dest, src, idxmap, S)
-            for S in vector_of_variables_types
-        ]
-        single_variable_not_added = [
-            copy_single_variable(dest, src, idxmap, S)
-            for S in single_variable_types
-        ]
+        # Order the copying of the variables by 1) their variable bridging cost 2) by starting with the vectors, as what was done before
+        # See issue #987
+        sorted_by_cost = sortperm(constraint_types; by=((F, S),) -> (MOI.get(dest, MOI.VariableBridgingCost{S}()), F == MOI.SingleVariable))
```

Comment:
In `constraint_types` you also have the constraints for which `F` is not `VectorOfVariables` or `SingleVariable`, you may want to filter that first.

Reply:
The cost should be `MOI.get(dest, MOI.VariableBridgingCost{S}()) - MOI.get(dest, MOI.ConstraintBridgingCost{F, S}())`.
If bridging a variable is cheap but bridging it as a constraint is even cheaper then it should not have priority.

Reply:
I think I get it now. I changed the cost and filtered `constraint_types`.